examples/set-label/.expected/diff.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/configs/example-config.yaml b/configs/example-config.yaml
+index 1c43349..02e8ab4 100644
+--- a/configs/example-config.yaml
++++ b/configs/example-config.yaml
+@@ -2,7 +2,11 @@ apiVersion: v1
+ kind: ConfigMap
+ metadata:
+   name: the-map
++  labels:
```

Comment:
Do tests currently check for idempotency?

Reply:
The test driver doesn't check it. However, the unit test covers the idempotency of the transformer and I have updated the tests to 
```
  - label_name: color
    label_value: orange
  - label_name: color
    label_value: orange
  - fieldSpecs:
    - kind: ConfigMap
      create: true
      path: data
    label_name: fruit
    label_value: apple
```

Reply:
@mengqiy This should be done by the e2e test harness.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/1396
 Will do it. But that doesn't block this PR.

Reply:
Yes, our test driver should cover idempotency.
Thanks for bring this up!